clients/scanoss/src/main/kotlin/model/Model.kt

```diff
@@ -35,21 +35,63 @@ data class KnowledgeBaseVersion(
     val daily: String
 )
 
+/**
+ * The “licenses” section of the RAW report.
+ */
 @Serializable
 data class License(
+    /** Name of the license */
     val name: String,
-    val source: String,
+
+    /** Location where the license was mined from. */
+    val source: Source,
+
+    /** Are there patent hints for this license.*/
     @SerialName("patent_hints")
-    val patentHints: String? = null,
-    val copyleft: String? = null,
+    @Serializable(with = BooleanSerializer::class)
+    val patentHints: Boolean? = null,
+
+    /** Is this considered a copyleft license or not. */
+    @Serializable(with = BooleanSerializer::class)
+    val copyleft: Boolean? = null,
+
+    /** URL of the OSADL checklist for this license. */
     @SerialName("checklist_url")
+
     val checklistUrl: String? = null,
     @SerialName("incompatible_with")
+
     val incompatibleWith: String? = null,
+
+    /** Date the OSADL data was last updated. */
```

Comment:
For readability, it would be nice to split out the documentation of existing properties and the addition of blank lines to a single preceeding commit, and then only do the type changes in this commit.